docs/guides/lender.md

```diff
@@ -1 +1,916 @@
-# Lender
\ No newline at end of file
+# Lender
+
+In this guide, you’ll learn how to build a simple defi dApp in under an hour using the [Código Platform](https://studio.codigo.ai). We will be building a dApp that will allow informal lenders to lend money.
+
+Código is an AI-powered code Generation Platform for blockchain developers and web3 teams that saves development time and increases the code's security.
+
+To get started, create an account by accessing [Código Studio](https://studio.codigo.ai) 
+
+## Create an account
```

Comment:
Instead of describing how to create an account redirect the user to the documentation that indicates how to create an account, it will be here `getting-started/installation/#código-studio` This way we don't need to maintain two or more places on how to create an account.